src/libstd/net/ip.rs

```diff
@@ -837,8 +838,16 @@ impl From<Ipv6Addr> for IpAddr {
 #[stable(feature = "rust1", since = "1.0.0")]
 impl fmt::Display for Ipv4Addr {
     fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
+        const IPV4_BUF_LEN: usize = 16; // Long enough for the longest possible IPv4 address
```

Comment:
Wouldn't 15 be enough? Is there a reason to round this up?

Reply:
Probably unconscious preference for powers of 2, I'll bring it down to 15